docs/howto/multiple-storage-backends.md

```diff
@@ -0,0 +1,260 @@
+
+## Working with Repositories  
+
+After setting up lakeFS Enterprise to connect with multiple storage backends, this section explains how to use these 
+connected storages when working with lakeFS.
+
+With multiple storage backends configured, lakeFS repositories are now linked to a specific storage backend. Together with
+the repository's [storage namespace](../understand/model.md#concepts-unique-to-lakefs), this defines the exact location in
+the underlying storage where the repository's data is stored.
+
+The choice of storage backend impacts the following lakeFS operations:
+
+### Creating a Repository
+
+In a multi-store setup, users must specify a storage ID when creating a repository. This can be done using the following methods:
+* API: Use the `storage_id` parameter in the [Create Repository endpoint](https://docs.lakefs.io/reference/api.html#/repositories/createRepository).  
+* CLI: Use the `storage-id` flag with the [repo create](../reference/cli.md#lakectl-repo-create) command:
+```bash
+lakectl repo create lakefs://my-repo s3://my-bucket --storage-id my-storage
+```
+**Note**: The `--storage-id` flag is currently hidden in the CLI. 
+* UI: Select a storage backend from the dropdown menu.
+![create repo with storage id](../assets/img/msb/msb_create_repo_ui.png)
+* High-level Python SDK: TODO
+
+**Important notes:**
+* In multi-store setups where a storage backend is marked as `backward_compatible = true`, repository creation requests
+without a storage ID will default to this storage.
+* If no storage backend is marked as `backward_compatible`, repository creation requests without a storage ID will fail.
+* Each repository is associated with a single backend and uses a single location.
+
+### Viewing Repository Details
+
+To check which storage backend is associated with a repository:
+* API – The [List Repositories](https://docs.lakefs.io/reference/api.html#/repositories/listRepositories) response includes the storage ID.
+* UI – The storage ID is displayed in the repository settings page.
+![repo settings](../assets/img/msb/msb_repo_settings_ui.png)
+
+### Importing Data into a Repository
+
+Importing data into a repository is supported when the following conditions are met:
+* The credentials used for the repository's backing blockstore allow access to the storage location.
+* The storage location is in the same region as the repository's backend.
+
+## Limitations 
+
+### Unsupported storage backends
+
+Multi-storage backend support has been validated on:
+* Self-managed S3-compatible object storage (e.g., MinIO)
+* Amazon S3
+* Local storage
+
+While this feature is designed to support any blockstore combination, testing for Azure and GCS in multi-store setups is 
+still in progress.
+
+{: .note}
+> **Note:** Other untested combinations may still work. You are encouraged to try them and share feedback.
```

Comment:
I would replace the encouraged to try them, prefer something like if you are interested please contact us. 
No one can really try it

Reply:
good call, done

Reply:
Strongly agree